components/eamxx/docs/developer/style/variables.md

```diff
@@ -0,0 +1,30 @@
+# Variables
+
+## Naming
+
+- Please name variables as ***descriptive nouns***.
+    - That is, for a variable holding data related to *horizontal winds*, choose
+      `horizontal_winds` or even `horiz_winds` versus `wh`.
+    - In cases for which this may be untenable, add comments explaining
+      non-obvious variable names.
+- In general, variables should use `snake_case` and be entirely lowercase,
+  unless capitalizing is a standard convention (e.g., `T` for temperature).
+- We do not currently employ a stylistic distinction between different types of
+  variables--e.g., `const`, `static`, member variables, etc.
+    - To that end, some developers use a non-standardized convention of
```

Comment:
I would strongly encourage the reader to embrace one of them though. It does not have to be uniform across the code, but each class should adopt it, unless it's a very small struct. The reason is that when reading a long method, where maybe an input and member have a similar name, you want to know which var represents the obj state:
```
void A::update_time (double time2) 
{
   blah blah code
   more blah blah code
   auto delta = some_func(time); // wait, was time a local var or the class member?
   even more blah blah code
}
```